devicon.json

```diff
@@ -227,6 +227,22 @@
         "color": "#002A41",
         "aliases": []
     },
+    {
+        "name": "bash",
+        "tags": [
+            "shell",
+            "command"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "plain"
+            ],
+            "font": []
```

Comment:
We need a font version of the icon to fulfill our https://github.com/devicons/devicon/blob/master/CONTRIBUTING.md guidelines.